src/main/java/lotto/TicketCollection.java

```diff
@@ -0,0 +1,41 @@
+package lotto;
+
+import view.ResultView;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class TicketCollection {
+    private int count;
+    private final List<LottoTicket> tickets = new ArrayList<>();
+    private final RandomPicker randomPicker = new RandomPicker();
+    public static final List<Integer> PRIZE = new ArrayList<>(Arrays.asList(0, 0, 0, 5000, 50000, 1500000, 2000000000));
```

Comment:
```suggestion
    public static final List<Integer> PRIZE = Arrays.asList(0, 0, 0, 5000, 50000, 1500000, 2000000000);
```

new ArrayList를 한번 더 감싸주지 않으셔도 괜찮습니다.

`Arrays.asList`

```java
    @SafeVarargs
    @SuppressWarnings("varargs")
    public static <T> List<T> asList(T... a) {
        return new ArrayList<>(a);
    }
```